components/reddit/actions/list-subreddits/list-subreddits.mjs

```diff
@@ -0,0 +1,107 @@
+import reddit from "../../reddit.app.mjs";
+import get from "lodash/get.js";
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "action",
+  key: "reddit-action-list-subreddits",
+  version: "0.0.1",
+  name: "List Subreddits",
```

Comment:
Do you think we should name this action "List Subreddits by Query" instead so its effect is clearer, and to differentiate it from potential future "List Subreddits <by/in X>" actions?

And if we do change the action `name`, also change the `key` to follow the [`app_name`_`slug-slugified-component-name` pattern](https://pipedream.com/docs/components/guidelines/#component-key-pattern).

Reply:
Yeah, you're right :)